src/modules/ie-objects/ie-objects.types.ts

```diff
@@ -313,6 +313,10 @@ export interface IeObjectsSitemap {
 	updatedAt: string;
 }
 
+export interface InewspaperTitle {
```

Comment:
We don't follow the I for interface convention in these repos
```suggestion
export interface NewspaperTitle {
```

![image](https://github.com/viaacode/hetarchief-proxy/assets/1710840/62939c6f-1f8c-47ac-b076-3927825ac043)